package.json

```diff
@@ -47,6 +47,7 @@
     "postinstall:warning": "echo '\n\nWARNING\n\nNot all node modules were installed because NODE_ENV is set to \"production\".\nThis should only be set after installing dependencies with \"yarn\". The app will not work.\n\n'"
   },
   "dependencies": {
+    "@babel/eslint-parser": "^7.18.2",
```

Comment:
- If we are upgrading, shouldn't we remove `"babel-eslint": "^10.0.1",` from the list?

:small_blue_diamond: Unless absolutely necessary for this PR, we can also exclude this this change since David's PR already covers the same thing.

p/s: please fix the nagging lint issues